include/class.import.php

```diff
@@ -189,7 +189,7 @@ function next() {
             foreach ($this->headers as $h => $label) {
                 $f = $this->fields[$h];
                 $f->_errors = array();
-                $T = $f->parse($csv[$i]);
+                $T = trim($f->parse($csv[$i]));
```

Comment:
Why not trim the csv content before passing it to the field to parse? 

```php
$T = $f->parse(trim($csv[$i]));
```

My thinking is field might fail parsing - should extra space fail field validation.